app/api/ddi-index-api/cdo.js

```diff
@@ -0,0 +1,20 @@
+const { post } = require('./base')
+const createCdoSchema = require('../../schema/ddi-index-api/cdo/create')
+
+const cdoEndpoint = 'cdo'
+
+const createCdo = async (cdo) => {
+  const { value, error } = createCdoSchema.validate(cdo)
+
+  if (error) {
+    throw error
+  }
+
+  const res = await post(cdoEndpoint, value)
```

Comment:
A general point for later, but presumably we'll need to pass some user info through to the API (or bearer token to be decrypted by the back-end) so we can record who created the record.

Reply:
Yeah good point - Will be needed for audit / event logging